core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -517,13 +535,15 @@ public FileInfo setAclEntries(List<String> acl) {
     }
 
     AccessControlList tAcl = AccessControlList.fromStringEntries(mOwner, mGroup, mAclEntries);
+    AccessControlList defaultAcl = AccessControlList.fromStringEntries(mOwner,
+        mGroup, mDefaultAclEntries);
 
     alluxio.thrift.FileInfo info =
         new alluxio.thrift.FileInfo(mFileId, mName, mPath, mUfsPath, mLength, mBlockSizeBytes,
         mCreationTimeMs, mCompleted, mFolder, mPinned, mCacheable, mPersisted, mBlockIds,
         mInMemoryPercentage, mLastModificationTimeMs, mTtl, mOwner, mGroup, mMode,
         mPersistenceState, mMountPoint, fileBlockInfos, TtlAction.toThrift(mTtlAction), mMountId,
-        mInAlluxioPercentage, mUfsFingerprint, tAcl.toThrift());
+        mInAlluxioPercentage, mUfsFingerprint, tAcl.toThrift(), defaultAcl.toThrift());
```

Comment:
does `fromThrift` have to be updated too? (and a unittest for that)

Reply:
done